friends/src/App.js

```diff
@@ -0,0 +1,102 @@
+import React, { Component } from 'react';
+import {Route} from 'react-router-dom';
+import axios from 'axios';
+import './App.css';
+import FriendsList from './components/FriendsList';
+import AddFriend from './components/AddFriend';
+import Friend from './components/Friend';
+
+class App extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+        friends: [],
+    };
+  }
+
+  componentDidMount() {
+    axios
+        .get('http://localhost:5000/friends')
+        .then(res => {
+            console.log('Response', res.data)
+            this.setState(() => ({ friends: res.data }));
+            
+        })
+        .catch(err => {
+        console.log('Server Error', err);
+        });
+  }
+
+  saveFriend = props => {
+    console.log('saveFriend props' , props)
+    const newFriend = {
```

Comment:
if your props is only having `name, age and email` you can pass that directly to your post